samples/nrf9160/gps/src/supl_support.c

```diff
@@ -36,13 +40,26 @@ int open_supl_socket(void)
 		.ai_canonname = NULL,
 	};
 
-	err = getaddrinfo(SUPL_SERVER, NULL, &hints, &info);
-	if (err) {
-		printk("Failed to resolve hostname %s on IPv4, errno: %d)\n",
-			SUPL_SERVER, errno);
-		return -1;
-	}
+	/* Try getaddrinfo many times, sleep a bit between retries */
+	do {
+		err = getaddrinfo(SUPL_SERVER, NULL, &hints, &info);
+		gai_cnt++;
+
+		if (err) {
+			printk("Failed to resolve hostname %s on IPv4, errno: %d)\n",
```

Comment:
You should only report this when you return -1. Not every time getaddrinfo fails

Reply:
done